.releaserc

```diff
@@ -9,7 +9,8 @@
       "assets": "false"
     }],
     ["@semantic-release/exec", {
-      "successCmd": "echo \"SEMVER_VERSION=${nextRelease.version}\" >> $GITHUB_ENV"
+      "successCmd": "echo \"SEMVER_VERSION=${nextRelease.version}\" >> $GITHUB_ENV",
+      "publishCmd": "./ci/trigger-catalog-onboarding-pipeline.sh --version=v${nextRelease.version}"
```

Comment:
hmm, in theory this should work, but I noticed in other DAs we have them split like this:
```
    ["@semantic-release/exec", {
      "successCmd": "echo \"SEMVER_VERSION=${nextRelease.version}\" >> $GITHUB_ENV"
    }],
    ["@semantic-release/exec",{
      "publishCmd": "./ci/trigger-catalog-onboarding-pipeline.sh --version=v${nextRelease.version}"
    }]
```

Reply:
ok I copied this from your internal doc, I'll open a PR to update the doc as well.

Reply:
I tried to update the internal doc but I don't have the permissions, anyway I fixed our code.

Reply:
thanks, I actually what you had would work too